buster.Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM mcr.microsoft.com/oryx/build:github-actions-buster-20210225.2
```

Comment:
What will be a sample workflow looks like for a user to build their apps in a buster environment? We have this buster.Dockerfile, but how can user use it, should we provide a parameter